plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -441,7 +464,8 @@ function PlannerOverlay:init()
             end,
         },
         widgets.Label{
-            frame={b=3, l=17},
+            frame={b=2, l=25},
+            text_pen=dfhack.pen.parse{fg=COLOR_DARKGREY},
```

Comment:
I like the color, but the positioning seems..distracting:
![image](https://user-images.githubusercontent.com/977482/229651425-79ecb98a-7da4-4a9e-a3c9-360a6202113d.png)

The right alignment of the `:` I get, but the `z:` sticking out on the left, not aligned with the line above or the line below just below it is strange. Not sure what the best solution is, though.

Reply:
i... 🤔 hm.
yeah, i'll see how to handle this tomorrow, alongside all of the other things!

Reply:
![image](https://user-images.githubusercontent.com/5853667/230059271-e0f26d0c-94fa-4b6f-9904-daa2958c7ca7.png)
now it's aligned with the `z:`